README.md

```diff
@@ -203,6 +203,19 @@ export interface DotenvLoaderOptions {
    */
   separator?: string;
 
+  /**
+   * If set, this function will transform all environment variable keys prior to parsing.
+   *
+   * Be aware: If you transform multiple keys to the same value only one will remain!
+   *
+   * @example
+   *
+   * .env file: `PORT=8080` and `keyTransformer: key => key.toLowerCase()` results in `{"port": 8080}`
+   *
+   * @param key environment variable key
+   */
+  keyTransformer?: (key: string) => string;
```

Comment:
I have added a description of the `keyTransformer` to _passing options_ in the readme. What other changes would you like me to make?